source/process/terminology.md

```diff
@@ -1,6 +1,26 @@
 # Terminology 
 
-Mattermost terminology used across the project. 
+Designing world-class software means bringing people together across disciplines and cultures. We want to create a limited amount of shared terminology to help us work better together, while being careful not to make it difficult for newcomers to follow our conversation. 
+
+Perhaps in future we'll have a bot that helps teach newcomers about the terminology in-context. Until then we have this guide. 
+
+## Purpose
+
+We use Mattermost terminology to achieve specific benefits: 
+
+#### 1) to increase precision through unambiguous definitions of useful terms
+
+For example, "0/5" and "5/5" help convey the level of conviction behind an opinion. Also, a precise classification of tickets as "Bug" or "Improvement" is critical since it affects scheduling and decision making, and so forth. 
+
+#### 2) to increase the speed of communication via a small number of frequently used acronyms 
+
+LHS and RHS are examples of a very limited number of acronyms to use to speed discussions, specifications, and ticket writing. 
```

Comment:
Propose adding in brackets what LHS and RHS means (left-hand sidebar and right-hand sidebar), otherwise this sentence doesn't make sense to someone who hasn't read the below list of terms.